en/controllers/middleware.rst

```diff
@@ -45,11 +70,11 @@ hook method will be called early in the request process, you can use the
 
     class Application extends BaseApplication
     {
-        public function middleware($middlewareStack)
+        public function middleware($middleware)
```

Comment:
https://github.com/cakephp/app/blob/master/src/Application.php#L37

We currently name the parameter `$middlewareQueue` in the app skeleton.